core/string-format.cpp

```diff
@@ -318,41 +318,43 @@ static QString formatPlotInfoInternal(const dive *d, const plot_info *pi, int id
 	const struct plot_data *entry = pi->entry + idx;
 
 	depthvalue = get_depth_units(entry->depth, NULL, &depth_unit);
-	put_format_loc(&b, translate("gettextFromC", "@: %d:%02d\nD: %.1f%s\n"), FRACTION(entry->sec, 60), depthvalue, depth_unit);
+	put_format_loc(&b, "%s: %d:%02d\n", translate("gettextFromC", "@"), FRACTION(entry->sec, 60));
+	put_format_loc(&b, "%s: %.1f%s\n", translate("gettextFromC", "D"), depthvalue, depth_unit);
```

Comment:
I wonder if I agree with your intent here...
this creates new translation strings `"@"` and `"D"` which are completely context free and hard to understand for translators. While the existing translation string is weird, at least that's something they can easily find in the sources and have all already translated...

I hear you regarding potential crashes... but as it is I'd find this rather hostile for translators.

I don't think I've played with the translation comment feature to see if that gets surfaced in Transifex. The idea would be to follow the translate call with a magic comment like this
```
put_format_loc(&b, "%s: %d:%02d\n", translate("gettextFromC", "@"), FRACTION(entry->sec, 60));
//: this designates the time shown in the tooltip - should be a single character
```

I'll need to play with that before I'll agree with this change.

Reply:
But one thing should be clear - this will create a noticeable amount of work for the translators. On strings that right now all work just fine. Even if I agreed with the underlying logic here - that's something I try to be very careful about.